deepmd/pt/model/model/__init__.py

```diff
@@ -105,6 +108,62 @@ def get_spin_model(model_params):
     return SpinEnergyModel(backbone_model=backbone_model, spin=spin)
 
 
+def get_linear_model(model_params):
+    model_params = copy.deepcopy(model_params)
+    weights = model_params.get("weights", "mean")
+    list_of_models = []
+    ntypes = len(model_params["type_map"])
+    for sub_model_params in model_params["models"]:
```

Comment:
would be better if we can use get_standard_model to parse sub-models.

Reply:
Why not use `get_model` directly?

Reply:
The sub-models are at the atomic level, but as far as I understand, the `get_standard_model` method return the wrapped model directly.

Reply:
I believe that #4186 requires model-level sub-models. In detail, she wants to combine the DP model and a DMFF wrapper model, which should not be at the atomic level.

Reply:
> I believe that #4186 requires model-level sub-models. In detail, she wants to combine the DP model and a DMFF wrapper model, which should not be at the atomic level.

Do we want to keep atomic level linear models and create a new class to hold model level linear models?